Sources/GRPC/GRPCServerRequestRoutingHandler.swift

```diff
@@ -216,25 +216,55 @@ extension GRPCServerRequestRoutingHandler: ChannelInboundHandler, RemovableChann
     }
   }
 
+  /// A call URI split into components.
+  struct CallPath {
+    /// The name of the service to call.
+    var service: String.UTF8View.SubSequence
+    /// The name of the method to call.
+    var method: String.UTF8View.SubSequence
+
+    /// Charater used to split the path into components.
+    private let pathSplitDelimiter = UInt8(ascii: "/")
+
+    /// Split a path into service and method.
+    /// Split is done in UTF8 as this turns out to be approximately 10x faster than a simple split.
+    /// URI format: "/package.Servicename/MethodName"
+    init?(requestURI: String) {
+      var utf8View = requestURI.utf8[...]
+      guard utf8View.splitFirst(separator: self.pathSplitDelimiter)?.count == 0 else {
```

Comment:
nit: `isEmpty` over `count == 0`

Reply:
I'm not sure you'll like my change here - I'm not sure I like it either but however you express this it's a little ugly.